acceptance_tests/features/view_reporting_unit_details.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Display reporting unit page
+  As an internal user
+  I need to to be able to view reporting unit details
+  So that I can undertake a number of potential actions related to that reporting unit
+
+  Background: Internal user is already signed in
+    Given the internal user is already signed in
+
+  @us044_s01
```

Comment:
Is this the decided way to go now?

Reply:
Not right now but we can come to a decision in the future and align them
Don't think they're getting used anywhere atm